package-system/AWSNativeSDK/build_AWSNativeSDK_windows.cmd

```diff
@@ -0,0 +1,54 @@
+@echo off
+REM 
+REM Copyright (c) Contributors to the Open 3D Engine Project.
+REM For complete copyright and license terms please see the LICENSE at the root of this distribution.
+REM 
+REM SPDX-License-Identifier: Apache-2.0 OR MIT
+REM 
+
+SET SRC_PATH=temp\src
+SET BLD_PATH=temp\build
+
+SET CMAKE_COMMON_ARGS=""
+call::GenerateCommonCMakeArgs CMAKE_COMMON_ARGS
+
+REM Debug Shared
+call:ConfigureAndBuild Debug Shared
+
+REM Debug Static
+call:ConfigureAndBuild Debug Static
+
+REM Release Shared
+call:ConfigureAndBuild Release Shared
+
+REM Release Static
+call:ConfigureAndBuild Release Static
+
+ECHO "Custom Build for AWSNativeSDK finished successfully"
+exit /b 0
+
+:ConfigureAndBuild
+SET BUILD_TYPE=%~1
+SET LIB_TYPE=%~2
+SET BUILD_SHARED=OFF
+IF %LIB_TYPE% EQU Shared (
+    SET BUILD_SHARED=ON
+)
+ECHO "CMake Configure %BUILD_TYPE% %LIB_TYPE% with %CMAKE_COMMON_ARGS%"
+call cmake -S %SRC_PATH% -B %BLD_PATH%\%BUILD_TYPE%_%LIB_TYPE% %CMAKE_COMMON_ARGS% -DBUILD_SHARED_LIBS=%BUILD_SHARED% -DCMAKE_BUILD_TYPE="%BUILD_TYPE%" -DCMAKE_INSTALL_BINDIR="bin/%BUILD_TYPE%" -DCMAKE_INSTALL_LIBDIR="lib/%BUILD_TYPE%"
+IF %ERRORLEVEL% NEQ 0 (
+    ECHO "CMake Configure %BUILD_TYPE% %LIB_TYPE% failed"
+    exit /b 1
```

Comment:
Note that this line will only exit the current ConfigureAndBuild function but not the entire process. I think you may want to terminate the execution of this batch when error happens at any time. Same for the exit call inside other functions.

Reply:
sounds good, will update